apisix/discovery/nacos.lua

```diff
@@ -165,7 +165,7 @@ local function fetch_instance(service)
     local json_str = res.body
     local data, err = core.json.decode(json_str)
     if not data then
-        log.error("get instance from nacos for " .. service  .. ", invalid response body: ", json_str, " err: ", err)
+        log.error("get instance for "..service..",invalid response body: ",json_str, " err: ", err)
```

Comment:
that is not a good error log style.
we need to avoid to concat string object, bad performance

Reply:
help 
could you help correct it and add some test cases， it’s too hard for me